docs/source/start.rst

```diff
@@ -14,8 +17,8 @@ MacOS or Linux, you can open `Terminal` and run the command.
 The information in the sections below is more advanced and not needed for the
 majority of the ASReview LAB users.
 
-Command line arguments for starting ASReview LAB
-------------------------------------------------
+Command line arguments
+----------------------
```

Comment:
```suggestion
Command line arguments for starting ASReview LAB
------------------------------------------------
```

This should not be too generic as it also matches other interfaces. This is derived from labelstudio.

Reply:
ok!